xxhash.h

```diff
@@ -3584,7 +3561,7 @@ XXH3_len_129to240_64b(const xxh_u8* XXH_RESTRICT input, size_t len,
 
     {   xxh_u64 acc = len * XXH_PRIME64_1;
         int const nbRounds = (int)len / 16;
-        int i;
+        short i;
```

Comment:
In many modern cpu architectures, `short` is a less efficient integer format than `int` (speedwise).